_data/roadmap.yml

```diff
@@ -78,32 +78,28 @@
     - name: 0MQ/ZeroMQ
       date: September, 2017
       status: completed
-    - name: Fluffy blocks
-      date:
-      status: ongoing
-    - name: GUI port to android
-      date:
-      status: ongoing
-    - name: Forum Funding System redesign
-      date:
-      status: ongoing
     - name: Subaddresses
-      date:
-      status: ongoing
+      date: October, 2017
+      status: completed
     - name: Multi-signatures (multisig)
+      date: December, 2017
+      status: completed
+- year: 2018
+  accomplishments:
+    - name: Forum Funding System redesign
       date:
       status: ongoing
-    - name: Kovri alpha release
+    - name: More efficient range proofs for RingCT to reduce transaction sizes
```

Comment:
Are we talking about Bulletproofs? Might be worth mentioning it in this item.

Reply:
I just moved the old items over, but yes I think that it references bulletproofs.